Doc/library/traceback.rst

```diff
@@ -86,10 +86,9 @@ The module defines the following functions:
 
 .. function:: print_last(limit=None, file=None, chain=True)
 
-   This is a shorthand for ``print_exception(sys.last_type, sys.last_value,
-   sys.last_traceback, limit, file, chain)``.  In general it will work only
-   after an exception has reached an interactive prompt (see
-   :data:`sys.last_type`).
+   This is a shorthand for ``print_exception(sys.last_exc, limit, file,
```

Comment:
I considered whether the change in the 'shorthand' needs a change note and decided not.  The four parameter call has worked since 3.10 and the older 6 parameter call still works, as explained in the much longer print_exception entry.  In other words, there have been and still are 2 equivalent shorthand descriptions and the (sensible) change here is to switch to the shorter and now more appropriate version.